lobster.sh

```diff
@@ -663,6 +664,8 @@ EOF
     if [ "$player" = "mpv" ] && ! command -v mpv >/dev/null; then
         if command -v mpv.exe >/dev/null; then
             player="mpv.exe"
+        elif echo "$(uname -a)" | grep -q "ndroid" 2>/dev/null; then
```

Comment:
do this here:
https://github.com/justchokingaround/lobster/blob/main/lobster.sh#L47-L50